your_child_workflow/your_child_workflow_dacx.py

```diff
@@ -0,0 +1,55 @@
+from temporalio import workflow
+from activities import ComposeGreetingInput
+from temporalio.workflow import ParentClosePolicy
+
+"""dacx
+To spawn a Child Workflow Execution in Python, use the [`execute_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#execute_child_workflow) function which starts the Child Workflow and waits for completion or
+use the [`start_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#start_child_workflow) function to start a Child Workflow and return its handle. 
+This is useful if you want to do something after it has only started, or to get the Workflow/Run ID, or to be able to signal it while running.
+
+:::note
+
+`execute_child_workflow()` is a helper function for `start_child_workflow()` plus `await handle`.
+
+:::
+dacx"""
+
+"""dacx
+Set the `parent_close_policy` parameter inside the [`start_child_workflow`](https://python.temporal.io/temporalio.workflow.html#start_child_workflow) function or the [`execute_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#execute_child_workflow) function to specify the behavior of the Child Workflow when the Parent Workflow closes.
+dacx"""
+
+
+@workflow.defn
+class ComposeGreeting:
+    @workflow.run
+    async def run(self, input: ComposeGreetingInput) -> str:
+        return f"{input.greeting}, {input.name}!"
+
+
+@workflow.defn
+class GreetingWorkflow:
+    @workflow.run
+    async def run(self, name: str) -> str:
+        return await workflow.execute_child_workflow(
+            ComposeGreeting.run,
+            ComposeGreetingInput("Hello", name),
+            id="hello-child-workflow-workflow-child-id",
+            parent_close_policy=ParentClosePolicy.TERMINATE,
```

Comment:
This is the default. We don't usually encourage people to explicitly set already-assumed default. May want to separate executing default child workflows with parent close policy.

Also, not sure `parent_close_policy` deserves an entire code sample for setting a field any more than `cancellation_type`, `id_reuse_policy`, or any other child workflow option.

Reply:
Can you confirm that the default child workflow snippet in docs won't show parent close policy as set? This seems to be shared sample code. We have terminate as the default because it's what we recommend. Changing the default should only be in the docs for this specific option.